person.rb

```diff
@@ -1,21 +1,39 @@
-class Person
-  def initalalize(age, parent_permission: true, name: 'Unknown')
+require './nameable'
+require './capitalize'
+require './trimmer'
+
+class Person < Nameable
+  def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    super()
   end
 
   attr_reader :id
   attr_accessor :name, :age
```

Comment:
# Congratulations:
- 
![Capture](https://user-images.githubusercontent.com/63972670/194025597-99466c8a-aa80-42c5-bde5-2a939d935645.JPG)

Reply:
Thank you